### PR TITLE
Add TiDB community upgrade

### PR DESCRIPTION
**Pull request by dcalvin**
This PR added the blog for TiDB Community Upgrade.
Translated from https://pingcap.com/blog-cn/tidb-community-upgrade/

### CODE REVIEW

File: tidb-community-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+---
+title: New Structure, New Roles - TiDB Community Upgrade!
+author: ['Jian Zhang']
+date: 2019-10-30
+summary: Four years of development has seen the TiDB community gradually grow into maturity. However, as the community expands, some shortcomings begin to loom out of the current community structure. After a series of reflections and summaries, we decided to upgrade and adjust the current community organization structure, with more community roles and community organizations introduced to better stimulate the vitality of the community and maintain a positive and healthy community environment.
```
Comment (CaitinChen) [inline tidb-community-upgrade.md]:
```suggestion
summary: PingCAP has upgraded and adjusted the TiDB community organization structure, with more community roles and community organizations introduced to better stimulate the vitality of the community and maintain a positive and healthy community environment.
```

Comment (CaitinChen) [reply tidb-community-upgrade.md]: I think this summary can be shorter so that it can be displayed completely in the link preview.

**Comment by dcalvin** [comment]
@CaitinChen Updated. PTAL again. Thanks.

File: tidb-community-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+    - Demonstrate good judgment and sense of responsibility for the health of the subprojects
+    - Set up the technical direction directly or by delegating the responsibilities
+    - Make or approve design decisions for the subprojects
+
+It can be seen in the old community structure that the growing TiDB user community that has been essential for product polishing and upgrading was obscured, and that the responsibilities for the roles in the developer community and the relationship between the roles are relatively simple. Therefore, we decided to include the TiDB user community in the new community structure while providing the TiDB developer community with clearly defined roles, rights, and responsibilities. At the same time, some new roles and new organizations have been introduced. Let's take a closer look.
+
+## New community structure
+
+### Change 1：Include the TiDB user community
```
Comment (CaitinChen) [inline tidb-community-upgrade.md]:
```suggestion
### Change 1: Include the TiDB user community
```

File: tidb-community-upgrade.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+* Committer: Recommended by Maintainer or Project Management Committee (PMC) as a Contributor who has made a significant contribution to TiDB. Must have completed at least one feature or fixed a major bug.
+
+* Maintainer: The planner and designer of the project, with the right to merge branches into the master, generated from Committer. A Maintainer should:
+
+    - Demonstrate good judgment and sense of responsibility for the health of the subprojects
+    - Set up the technical direction directly or by delegating the responsibilities
+    - Make or approve design decisions for the subprojects
+
+It can be seen in the old community structure that the growing TiDB user community that has been essential for product polishing and upgrading was obscured, and that the responsibilities for the roles in the developer community and the relationship between the roles are relatively simple. Therefore, we decided to include the TiDB user community in the new community structure while providing the TiDB developer community with clearly defined roles, rights, and responsibilities. At the same time, some new roles and new organizations have been introduced. Let's take a closer look.
```
Comment (dcalvin) [inline tidb-community-upgrade.md]:
```suggestion
The TiDB user community has played an important role to play in product polishing and upgrading. However, it was obscured in the old community structure. In addition, the responsibilities for the roles in the developer community and the relationship between roles are relatively simple. Therefore, we decided to include the TiDB user community in the new community structure while providing the TiDB developer community with clearly defined roles, rights, and responsibilities. At the same time, some new roles and new organizations have been introduced. Let's take a closer look.
```

**Pull request merged**